changelogs/fragments/9-semantic-markup-roles.yml

```diff
@@ -0,0 +1,4 @@
+minor_changes:
+  - "Add support for semantic markup in roles (https://github.com/ansible-community/antsibull-docs-parser/pull/9)."
+breaking_changes:
```

Comment:
Isn't adding `entrypoint` to `OptionNamePart` and `ReturnValuePart` also a breaking change? It changes the order of positional arguments when instantiating the class and the option indexes in the tuple are different.

Reply:
Improved in 6ffdc1f7ece584fa2d85d4c9251eadeba931d6cf.